src/host/misc.h

```diff
@@ -44,6 +44,38 @@ int ConvertToOem(const UINT uiCodePage,
                  _Out_writes_(cchTarget) CHAR* const pchTarget,
                  const UINT cchTarget) noexcept;
 
+template<typename CharT, typename Traits>
+void StringToInputEvents(std::basic_string_view<CharT, Traits> str, std::deque<std::unique_ptr<IInputEvent>>& events)

```

Comment:
We should make sure that this is named more clearly. It doesn't create fully formed input events that are safe for everyone to consume - it only creates keydown events.